packages/firebase_database/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.15
```

Comment:
For a breaking change you should bump the version to 0.1.0. That way, people who have pinned their usage of this plugin to "^0.0.14" will not be auto-upgraded into a broken state.

Reply:
@goderbauer Thank you!